src/SlashCommands.tsx

```diff
@@ -141,13 +141,25 @@ export class Command {
 
     run(roomId: string, threadId: string, args: string) {
         // if it has no runFn then its an ignored/nop command (autocomplete only) e.g `/me`
-        if (!this.runFn) return reject(_t("Command error"));
+        if (!this.runFn) {
+            return reject(
+                newTranslatableError(
+                    "Command error: This command should have been handled by another handler and we" +
+                        "  can't try to run something without a command function.",
+                ),
+            );
+        }
 
         const renderingType = threadId
             ? TimelineRenderingType.Thread
             : TimelineRenderingType.Room;
         if (this.renderingTypes && !this.renderingTypes?.includes(renderingType)) {
-            return reject(_t("Command error"));
+            return reject(
+                newTranslatableError(
+                    "Command error: Unable to find rendering type (%(renderingType)s)",
+                    { renderingType },
+                ),
```

Comment:
Updated error string here so if it ever happens, there is more info to pass along.

It seems like for these type of errors, we could be a lot more proactive in reporting them. We expect this to never happen AFAICT. But if it does, we should automatically log it to our Sentry instance and tell the user, "Sorry, it looks like you hit an error but we've just reported it to the developers to let them know." wdyt?

Reply:
Sentry is opt-in in Labs, also

Reply:
We can punt this off to another PR.

What do you think about keying off the same Labs flag `automaticErrorReporting` and only report if enabled?